src/iguana/algorithms/clas12/ZVertexFilter/Algorithm.cc

```diff
@@ -38,19 +38,21 @@ namespace iguana::clas12 {
     particleBank.getMutableRowList().filter([this](auto bank, auto row) {
         auto zvertex = bank.getFloat("vz", row);
         auto pid = bank.getInt("pid", row);
-        auto accept  = Filter(zvertex,pid);
-        m_log->Debug("input vz {} pid {} -- accept = {}", zvertex, pid, accept);
+        auto status = bank.getInt("status", row);
```

Comment:
`status` is `short` type; `hipo::bank::getShort` actually returns an `int`, so downstream code shouldn't need any changes.
```suggestion
        auto status = bank.getShort("status", row);
```